org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/task/WorkspaceProjectsChangeListener.java

```diff
@@ -38,7 +39,8 @@ public WorkspaceProjectsChangeListener(TaskView taskView) {
 
     @Override
     public void onEvent(Event event) {
-        if (event instanceof ProjectCreatedEvent || event instanceof ProjectDeletedEvent || event instanceof GradleNatureAddedEvent) {
+        if (event instanceof ProjectCreatedEvent || event instanceof ProjectDeletedEvent || event instanceof GradleNatureAddedEvent
```

Comment:
Please add test coverage for the changes to the `TaskViewContentTest` class.

Reply:
Thanks for the review. I have made other 2 changes. But I have added a test case as well. But when I try to run the test I am getting following error.
![image](https://user-images.githubusercontent.com/20665940/37611787-a2e0343a-2ba3-11e8-8cfc-7d0af5f851c2.png)

Reply:
Judging from the icons of the `.groovy` files it seems that your Eclipse doesn't have Groovy tooling installed. Did you set up your [development environment](https://github.com/eclipse/buildship/blob/master/docs/development/Setup.md) via Oomph?

Reply:
Oh, I will install the Groovy plugins into Eclipse. 
Yes, I followed the same steps mentioned in the link.

But I will it a try once again. Give me some time. 
Thanks for helping me!!